docs/js-guide.md

```diff
@@ -3,9 +3,12 @@
 This document explains the tools and conventions that we use to maintain
 quality and consistency of our JavaScript code.
 
+The rationales for various conventions are documented briefly _in italics_.
```

Comment:
This makes for a fair amount of text in italic...I like the idea of having a "do this"..."because this" structure, though, that's good!